web/server/codechecker_server/api/report_server.py

```diff
@@ -662,6 +677,16 @@ def extended_data_db_to_api(erd):
         fileId=erd.file_id)
 
 
+def comment_data_db_to_api(comm):
+    return ttypes.CommentData(
+        id=comm.id,
+        author=comm.author,
+        message=comm.message.decode("utf-8").replace("\\", ""),
```

Comment:
For system comments we should post process the message. See:
https://github.com/Ericsson/codechecker/blob/47cf966edcafaaaf1047298d8fe0b03f0f3ca82a/web/server/codechecker_server/api/report_server.py#L1610-L1630
We can move this logic to a separate helper function and use it here and in the original place.

Reply:
I have done the changes. Kindly have a look. I am getting `%author%` and `%date` in the system comments. Is that okay??